packages/realm/src/assert.ts

```diff
@@ -63,6 +63,11 @@ assert.number = (value: unknown, target?: string): asserts value is number => {
   assert(typeof value === "number", () => new TypeAssertionError("a number", value, target));
 };
 
+assert.numericString = (value: unknown, target?: string): asserts value is number => {
```

Comment:
Is it intended for the value to be converted and used as a number after calling this function?

Reply:
Oh ... good catch. This was a copy-paste-o.
```suggestion
assert.numericString = (value: unknown, target?: string) => {
```

Reply:
Okay np 🙂  It may still be helpful to have `asserts value is string` instead though?

Reply:
I agree 👍 and pushed a commit with that.